### ISSUE TITLE
New tag additions not showing up from tag form?

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3176]
@derek-allen just checking in here, I noticed that when submitting a new tag (try on https://publiclab.org/notes/warren/07-25-2018/mapknitter-map-of-10-000-suns-in-providence), they don't appear as they used to upon submitting. 

They do disappear when deleted! Would you be able to take a peek at this and see if you can tell what's going on? Thank you!

**Comment by jywarren** [comment]
I think it's related to the presence or absence of this file? 

https://github.com/publiclab/plots2/blob/master/app/assets/javascripts/tagging.js

Or maybe the selectors that file uses?

**Comment by derek-allen** [comment]
Yeah, I'll take a look!

**Comment by jywarren** [comment]
thank you!!!

**Comment by derek-allen** [comment]
Yeah, the issue was with selectors. It should be fixed in the pull request i just opened up.

Interestingly, the tag form appears even when you aren't logged in, which might not be intended. Also, when a tag is added, its always colored like a power tag, but after a refresh goes to the proper place.

I reverted to commit 177b72e check, and this is consistent with the behavior pre-sidebar tagging, system but they might be good issues to flag if this isn't intended. Let me know if these issues should be created and I can take care of it!
